infra-l2-waf/template.yaml

```diff
@@ -0,0 +1,255 @@
+              - Name: GenericRFI_URIPATH
+              - Name: NoUserAgent_HEADER
+              - Name: RestrictedExtensions_QUERYARGUMENTS
+              - Name: RestrictedExtensions_URIPATH
+              - Name: SizeRestrictions_BODY
+              - Name: SizeRestrictions_Cookie_HEADER
+              - Name: SizeRestrictions_QUERYSTRING
+              - Name: SizeRestrictions_URIPATH
+              - Name: UserAgent_BadBots_HEADER
+          OverrideAction:
+            None: {}
+          VisibilityConfig:
+            SampledRequestsEnabled: true
+            CloudWatchMetricsEnabled: true
+            MetricName: !Sub '${AWS::StackName}-${Environment}-aws-crs-metric'
+
+        - Name: Bad-Inputs
+          Priority: 1
+          Statement:
+            ManagedRuleGroupStatement:
+              VendorName: AWS
+              Name: AWSManagedRulesKnownBadInputsRuleSet
+              ExcludedRules:
+              - Name: ExploitablePaths_URIPATH
+              - Name: Host_localhost_HEADER
+              - Name: JavaDeserializationRCE_BODY
+              - Name: JavaDeserializationRCE_HEADER
+              - Name: JavaDeserializationRCE_QUERYSTRING
+              - Name: JavaDeserializationRCE_URIPATH
+              # - Name: Log4JRCE
+              - Name: PROPFIND_METHOD
+          OverrideAction:
+            None: {}
+          VisibilityConfig:
+            SampledRequestsEnabled: true
+            CloudWatchMetricsEnabled: true
+            MetricName: !Sub '${AWS::StackName}-${Environment}-bad-inputs-metric'
+
+        - Name: Anonymous-IpList
+          Priority: 2
+          Statement:
+            ManagedRuleGroupStatement:
+              VendorName: AWS
+              Name: AWSManagedRulesAnonymousIpList
+              ExcludedRules: 
+              - Name: AnonymousIPList
+              - Name: HostingProviderIPList
+          OverrideAction:
+            None: {}
+          VisibilityConfig:
+            SampledRequestsEnabled: true
+            CloudWatchMetricsEnabled: true
+            MetricName: !Sub '${AWS::StackName}-${Environment}-anonymous-iplist-metric'
+
+        - Name: SQLInject-RuleSet
+          Priority: 3
+          Statement:
+            ManagedRuleGroupStatement:
+              VendorName: AWS
+              Name: AWSManagedRulesSQLiRuleSet
+              ExcludedRules:
+              - Name: SQLiExtendedPatterns_QUERYARGUMENTS
+              - Name: SQLi_BODY
+              - Name: SQLi_COOKIE
+              - Name: SQLi_QUERYARGUMENTS
+              - Name: SQLi_URIPATH
+          OverrideAction:
+            None: {}
+          VisibilityConfig:
+            SampledRequestsEnabled: true
+            CloudWatchMetricsEnabled: true
+            MetricName: !Sub '${AWS::StackName}-${Environment}-SQLinjection-ruleset-metric'
+
+        - Name: RateBased-CountIpRule
+          Priority: 4
+          Statement:
+            RateBasedStatement:
+              Limit: 100
```

Comment:
is this too low? AWS guidance is to start at 2000. Also note that we can't use this on endpoints called by our SSR FE or by another BE.